src/software/backend/input/network/network_filter.cpp

```diff
@@ -0,0 +1,392 @@
+#include "software/backend/input/network/network_filter.h"
```

Comment:
:exclamation: Copied from software/backend/input/network/networking with some refactoring and new functions